salt/utils/win_runas.py

```diff
@@ -1,34 +1,29 @@
 # -*- coding: utf-8 -*-
 '''
 Run processes as a different user in Windows
-
-Based on a solution from http://stackoverflow.com/questions/29566330
 '''
-from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, unicode_literals
 
 # Import Python Libraries
 import os
 import logging
 
 # Import Third Party Libs
+import psutil
```

Comment:
Achtung! This needs to either use the psutil compat lib in `salt.utils` or it needs to be conditionally imported.